js/src/widgets/metadataView.js

```diff
@@ -126,8 +126,8 @@
   getMetadataDetails: function(jsonLd) {
       // TODO: This should not default to English
       var mdList = {
-          'label':        '<b>' + jsonLd.label + '</b>' || '',
-          'description':  jsonLd.description || ''
```

Comment:
I think `description` can also be treated the same way as label, and therefore it is valid to call `getTextValue` on it as well, per: http://iiif.io/api/presentation/2.0/#language-of-property-values.